IPython/html/templates/page.html

```diff
@@ -106,8 +106,6 @@
   {% endblock %}
 </div>
 
-<div id="header-spacer"></div>
```

Comment:
You'll also want to delete the accompanying Javascript which sets the height of the spacer and all references to it (see `Page.prototype._resize_header` in `/static/base/js/page.js`).